packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -795,8 +795,16 @@ export default class CosmeticFilter implements IFilter {
       .slice(1)
       .map((part) => {
         if (
-          (part.startsWith(`'`) && part.endsWith(`'`)) ||
-          (part.startsWith(`"`) && part.endsWith(`"`))
+          (part.startsWith(`'`) &&
+            part.endsWith(`'`) &&
+            part.length >= 2 &&
+            !part.slice(1, -1).match(/[^\\]'/) &&
```

Comment:
Is there any difference than checking the first char code of the char at index of 1. `part.charCodeAt(1)`